modules/ROOT/pages/dw-core-functions-flatmap.adoc

```diff
@@ -3,42 +3,60 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-//* <<flatmap1>>
-//* <<flatmap2>>
-
 
 [[flatmap1]]
 == flatMap(Array<T>, (item: T, index: Number) -> Array<R>): Array<R>
 
-Maps an array of items using the specified `callback` and applies `flatten`
-to the resulting array.
+Applies a transformation function to each element in a list (array) and
+then flattens the result.
+
+
+Instead of returning an array of arrays (as `map` does), it returns a
+flattened array (see the `flatten` function).
+
+=== Parameters
+
+[%header, cols="1,3"]
+|===
+| Name   | Description
+| `items` | The list (`Array` type) to map.
+| `mapper` | The `item` and/or `index` in the list to flatten.
+|===
 
-.Transform
-[source,dataweave,linenums]
+=== Example
+
+This example returns an array containing each index and item in order. Note
+that you can also write the example like this:
+`{ "users" : ["joe", "pete", "matt"] flatMap ([$$ as String, $]) }`
```

Comment:
But you haven't shown me any other way to write the example yet, so the lead-in sentence is confusing.